xarray/tests/test_plot.py

```diff
@@ -1828,6 +1830,31 @@ def test_datetime_line_plot(self):
         self.darray.plot.line()
 
 
+@requires_nc_time_axis
+@requires_cftime
+class TestCFDatetimePlot(PlotTestCase):
+    @pytest.fixture(autouse=True)
+    def setUp(self):
+        '''
+        Create a DataArray with a time-axis that contains cftime.datetime64
+        objects.
+        '''
+        month = np.arange(1, 13, 1)
+        data = np.sin(2 * np.pi * month / 12.0)
+
+        darray = DataArray(data, dims=['time'])
+        darray.coords['time'] = xr.cftime_range(start='2017',
+                                                periods=12,
+                                                freq='1M',
+                                                calendar='noleap')
+
+        self.darray = darray
+
+    def test_cfdatetime_line_plot(self):
+        # test if line plot raises no Exception
+        self.darray.plot.line()
+
+
```

Comment:
It could also be helpful to have a test for the error behavior when nc-time-axis is not installed (you'll need to be precise with the skip logic, but it should be doable).

Reply:
I am a bit unsure how to even approach this, is there an examplel somewhere else that I could cheat of? My py.test knowledge is very limited.

Reply:
Sure -- I think you'll sort of want to do the opposite of what's done here for the skipping logic (i.e. skip if nc-time-axis *is* installed):
https://github.com/pydata/xarray/blob/27cf53f24c6c3b6ec05a435d3606ebba977b6343/xarray/tests/test_cftimeindex.py#L814
To test that the expected error is raised, you can use `raises_regex`, e.g.
https://github.com/pydata/xarray/blob/27cf53f24c6c3b6ec05a435d3606ebba977b6343/xarray/tests/test_plot.py#L1853-L1855

Reply:
Thanks a lot @spencerkclark, I think this works now. Just submitted a fix that for the linter and then it should pass all the tests.